scripts/combine_profiles.py

```diff
@@ -0,0 +1,348 @@
+        # Workaround
+        if struct == 'VkPhysicalDeviceVulkan11Features':
+            return 'VkPhysicalDeviceVulkan11Features'
+
+        version = None
+        if self.registry.structs[struct].definedByVersion:
+            version = self.registry.structs[struct].definedByVersion
+        else:
+            aliases = self.registry.structs[struct].aliases
+            for alias in aliases:
+                if registry.structs[alias].definedByVersion:
+                    version = registry.structs[alias].definedByVersion
+                    break
+        if version is None:
+            return False
+        return 'VkPhysicalDeviceVulkan' + str(version.major) + str(version.minor) + 'Features' if feature else 'Properties'
+
+    def add_struct(self, struct_name, struct, merged):
+        if struct_name in merged:
+            for member in struct:
+                merged[struct_name][member] = struct[member]
+        else:
+            merged[struct_name] = struct
+
+    def add_members(self, merged, entry):
+        for member in entry:
+            merged[member] = entry[member]
+
+    def find_higher_struct(self, struct1, struct2):
+        if registry.structs[struct1].definedByVersion:
+            return struct1
+        if registry.structs[struct2].definedByVersion:
+            return struct2
+        ext1_ext = False
+        ext1_other = False
+        ext2_ext = False
+        ext2_other = False
+        for ext in registry.structs[struct1].definedByExtensions:
+            if registry.extensions[ext].name[3:6] == 'EXT':
+                ext1_ext = True
+            else:
+                ext1_other = True
+        for ext in registry.structs[struct2].definedByExtensions:
+            if registry.extensions[ext].name[3:6] == 'EXT':
+                ext2_ext = True
+            else:
+                ext2_other = True
+        
+        if not ext1_ext and not ext1_other:
+            return struct1
+        if not ext2_ext and not ext2_other:
+            return struct2
+        if not ext1_other:
+            return struct1
+        if not ext2_other:
+            return struct2
+
+        return struct1
+
+    def get_promoted_version(self, vk_version):
+        if vk_version is None:
+            return None
+        version = vk_version[11:]
+        underscore = version.find('_')
+        major = version[:underscore]
+        minor = version[underscore+1:]
+        return [major, minor]
+
+    def get_profiles(self, profile_name, api_version, description, contributors):
+        profiles = dict()
+        profiles[profile_name] = dict()
+        profiles[profile_name]['version'] = 1
+        profiles[profile_name]['status'] = 'BETA'
+        profiles[profile_name]['api-version'] = api_version
+        profiles[profile_name]['label'] = 'Merged profile'
+        profiles[profile_name]['description'] = description
+        profiles[profile_name]['contributors'] = contributors
```

Comment:
I think we should rather leave the contributor section empty of only indicate the merge tool.

I am guessing that people would rather not want to be contacted or associated with a profiles they probably never heard of...